### ISSUE TITLE
Sharp edged corner labels stick out of the input field

### ISSUE THREAD

**Issue by brainscript** [Semantic-Org/Semantic-UI#4604]
![Screenshot](https://cloud.githubusercontent.com/assets/8706734/18965961/9fda1e42-867f-11e6-8fa3-5929a022d1f3.png)
As you can see on the screenshot, the **corners** of the `ui corner label` **stick out** of the input field.
That's hard to notice without adding a color, but if you do this really looks quite ugly ...

Is this supposed to happen? Or is it my mistake? :confused:
```html
<div class="field">
  <label>Enter some random example crap ...</label>
  <div class="ui corner labeled left icon input">
    <i class="idea icon"></i>
    <input id="example" type="text" name="example" placeholder="Example">
    <div class="ui red corner label" title="Unsecure example!">
      <i class="spy icon"></i>
    </div>
  </div>
</div>
```
If it's a bug, please fix it. It's such a pain in the eye to me ... :eyes:

**Comment by tabeth** [comment]
That's not a bug:

```
A corner label must be positioned inside a container with position: relative to display properly. If a container is rounded you will need to add overflow:hidden to the container to produce a rounded label.
```
[from the docs](http://semantic-ui.com/elements/label.html)

**Comment by brainscript** [comment]
@tabeth 
```sass
.ui.corner.labeled.field
  position: relative
```
**Like this?** It still looks the same ...
![Screenshot](https://cloud.githubusercontent.com/assets/8706734/18967111/4ea06522-8684-11e6-8e37-7af2162a49c3.png)

**Comment by brainscript** [comment]
Oh, I have to add the `overflow: hidden` to the corner label itself.

**Comment by tabeth** [comment]
Yeah, I was about to create a jsfiddle, but it sounds like you got it.

**Comment by brainscript** [comment]
But why is this not enabled by default for all corner labels inside of input fields (**They are rounded ...**)?

**Comment by tabeth** [comment]
That's a good point. If no one else gets to this I'll submit a PR for that.
